seed_services_client/service_rating.py

```diff
@@ -0,0 +1,88 @@
+
+"""
+Client for Service Rating Store HTTP services APIs.
+
+"""
+import requests
+import json
+
+
+class ServiceRatingApiClient(object):
+
+    """
+    Client for ServiceRating API.
+
+    :param str api_token:
+
+        An API Token.
+
+    :param str api_url:
+        The full URL of the API. Defaults to
+        ``http://seed-service-rating/api/v1``.
+
+    """
+
+    def __init__(self, api_token, api_url=None, session=None):
+        if api_url is None:
+            api_url = "http://seed-service-rating/api/v1"
+        self.api_url = api_url
+        self.headers = {
+            'Content-Type': 'application/json',
+            'Authorization': 'Token %s' % api_token
+        }
+        if session is None:
+            session = requests.Session()
+        session.headers.update(self.headers)
+        self.session = session
+
+    def call(self, endpoint, method, obj=None, params=None, data=None):
+        if obj is None:
+            url = '%s/%s/' % (self.api_url.rstrip('/'), endpoint)
+        else:
+            url = '%s/%s/%s/' % (self.api_url.rstrip('/'), endpoint, obj)
+        result = {
+            'get': self.session.get,
+            'post': self.session.post,
+            'patch': self.session.patch,
+            'delete': self.session.delete,
+        }.get(method, None)(url, params=params, data=json.dumps(data))
+        result.raise_for_status()
+        if method is "delete":  # DELETE returns blank body
+            return {"success": True}
+        else:
+            return result.json()
```

Comment:
This seems to be a method common to all the django rest framework APIs. Maybe we can factor it out into a base class, so that we don't have to repeat it?